support-frontend/assets/helpers/__tests__/checkoutsTest.ts

```diff
@@ -11,60 +13,88 @@ jest.mock('helpers/globalsAndSwitches/globals', () => ({
 	isSwitchOn: jest.fn(),
 }));
 
-const mock = (mockFn: any) => mockFn;
+const mock = (mockFn: unknown) => mockFn as jest.Mock;
+
```

Comment:
I'm not quite sure why we did this in the first place - effectively this function is just a typecast.

I've fixed it up to remove the errors, but it would also be possible to just replace all instances with:

```ts
(isSwitchOn as jest.Mock)
```

Reply:
Definitely quite gross to be having to mock like this in the first place though (we need more dep injection!)